src/types/chart/chart.ts

```diff
@@ -85,12 +85,13 @@ export interface TitleDesign {
   readonly color?: Color;
 }
 
-export type TrendType = "polynomial" | "exponential" | "logarithmic";
+export type TrendType = "polynomial" | "exponential" | "logarithmic" | "movingAverage";
```

Comment:
Rename the type in case we implement other types of moving averages in the future
```suggestion
export type TrendType = "polynomial" | "exponential" | "logarithmic" | "trailingMovingAverage";
```